### PR TITLE
v10 Conversion

### PR DESCRIPTION
**Pull request by EndlesNights**
Have not fully tested everything about, but get the module support for Foundry v10.

### CODE REVIEW

File: src/module.json [hunk 1]
```diff
@@ -56,9 +57,12 @@
 			"link": "https://raw.githubusercontent.com/ElfFriend-DnD/foundryvtt-5eOGLCharacterSheet/master/readme-img/biography.png"
 		}
 	],
-	"minimumCoreVersion": "9",
-	"compatibleCoreVersion": "9",
-	"minimumSystemVersion": "1.4.0",
+	"minimumSystemVersion": "2.0.0",
```
Comment (akrigline) [inline src/module.json]: i believe this has been replaced with something in `relationships`

**Comment by EndlesNights** [comment]
You're right, it should now be:
```
	"relationships": {
		"systems": [
			{
				"id": "dnd5e",
				"type": "system",
				"manifest": "https://raw.githubusercontent.com/foundryvtt/dnd5e/master/system.json",
				"compatibility": {
					"minimum": "2.0.0"
				}
			}
		]
	},
```

**Review by akrigline — APPROVED** [review]
I likewise haven't tested this, but will put out a release with this code if you don't object

**Comment by SirXen0** [comment]
Would love to test this, my players only use the OGL sheet and would be fine testing changes just to use the new foundry version.

**Comment by Someone2345** [comment]
At the risk of sounding like an idiot - is there a way to install and test this?
Many thanks

**Comment by mcabel** [comment]
hi, were the fixes here functional, or it ended up as buggy?

**Pull request merged**